### PR TITLE
#1971 zoom button behavior

### PR DESCRIPTION
**Pull request by blattmann**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,12 @@
+ifeq ($(OSTYPE),darwin14)
+OS ?= Darwin
+endif
+ifeq ($(OS),Darwin) 
+sed ?= /usr/local/Cellar/gnu-sed/4.2.2/bin/sed
```
Comment (sbrunner) [inline Makefile]: sed should be in capital

File: Makefile [hunk 2]
```diff
@@ -366,7 +379,7 @@ node_modules/angular/angular.min.js: .build/node_modules.timestamp
 .PRECIOUS: .build/examples-hosted/%.html
 .build/examples-hosted/%.html: examples/%.html
 	mkdir -p $(dir $@)
-	sed -e 's|\.\./node_modules/openlayers/css/ol.css|lib/ngeo.css|' \
+	/usr/local/Cellar/gnu-sed/4.2.2/bin/sed -e 's|\.\./node_modules/openlayers/css/ol.css|lib/ngeo.css|' \
```
Comment (sbrunner) [inline Makefile]: /usr/local/Cellar/gnu-sed/4.2.2/bin/sed => $(SED)
same bellow

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,12 @@
+ifeq ($(OSTYPE),darwin14)
+OS ?= Darwin
+endif
+ifeq ($(OS),Darwin) 
+sed ?= /usr/local/Cellar/gnu-sed/4.2.2/bin/sed
+else
+sed ?= sed
+endif
```
Comment (sbrunner) [inline Makefile]: I think that preferable to replace all this section by :
```make
SED ?= sed
```
and create your own makefile with:
```make
OS = Darwin
SED = /usr/local/Cellar/gnu-sed/4.2.2/bin/sed
include Makefile
```
Than you should do the make commands with:
`make -f <your_makefile> ...`

File: Makefile [hunk 4]
```diff
@@ -64,7 +73,11 @@ TX_GIT_BRANCH ?= master
 ifeq (,$(wildcard $(HOME)/.transifexrc))
 TOUCHBACK_TXRC = touch --date "$(shell date --iso-8601=seconds)" $(HOME)/.transifexrc
 else
-TOUCHBACK_TXRC = touch --date "$(shell stat -c '%y' $(HOME)/.transifexrc)" $(HOME)/.transifexrc
+ifeq ($(OS),Darwin)
+TOUCHBACK_TXRC = touch -t "$(shell stat -f '%m' -t "%Y%m%dT%H%M.%S" $(HOME)/.transifexrc)" $(HOME)/.transifexrc
+#else
+#TOUCHBACK_TXRC = touch --date "$(shell stat -c '%y' $(HOME)/.transifexrc)" $(HOME)/.transifexrc
```
Comment (sbrunner) [inline Makefile]: Thoes two lines are required for Linux users ...

**Comment by pgiraud** [comment]
Hi @blattmann, can you please rebase the branch for this pull request?

**Comment by blattmann** [comment]
PR will be closed. #631 includes the changed less file.

**Pull request closed without merging**